src/utils.js

```diff
@@ -148,6 +148,9 @@ function createAxiosError(message, config, response, code) {
   var error = new Error(message);
   error.isAxiosError = true;
   error.config = config;
+  error.request = {
+    responseUrl: config.url,
```

Comment:
Shouldn't this be `responseURL` and not `responseUrl`. I least in my browser `URL` is uppercased. (see https://github.com/ctimmerm/axios-mock-adapter/pull/287)